xonsh/environ.py

```diff
@@ -143,7 +145,13 @@ def default_value(f):
 def is_callable_default(x):
     """Checks if a value is a callable default."""
     return callable(x) and getattr(x, '_xonsh_callable_default', False)
-if ON_WINDOWS:
+
+if ON_CYGWIN:
+    DEFAULT_PROMPT = ('{env_name}'
+                      '{BOLD_INTENSE_GREEN}{user}@{hostname}'
+                      '{BOLD_INTENSE_CYAN} {cwd}{NO_COLOR}'
```

Comment:
At least for me, the `BLUE` color on Cygwin looks just fine.  So I might recommend borrowing from the POSIX prompt instead of the Windows prompt.

Also, can remove the center `{NO_COLOR}` at the end of the this line and `{BOLD_INTENSE_CYAN}` from the start of the next line and still be equivalent, I think.